### PR TITLE
minio: add launchd init script for running a daemon, revised

### PR DESCRIPTION
**Pull request by ghost**
add launchd init script for running minio as a daemon, revised
-get rid of unnecessary methods
-remove comment

### CODE REVIEW

File: Formula/minio.rb [hunk 1]
```diff
@@ -41,6 +41,42 @@ def install
     bin.install buildpath/"minio"
   end
 
+  DATADIR = "var/minio".freeze
+  CONFIGDIR = "etc/minio".freeze
```
Comment (MikeMcQuaid) [inline Formula/minio.rb]: Just inline these variables.

File: Formula/minio.rb [hunk 2]
```diff
@@ -41,6 +41,42 @@ def install
...
+      <key>KeepAlive</key>
+      <true/>
+      <key>Label</key>
+      <string>#{plist_name}</string>
+      <key>ProgramArguments</key>
+      <array>
+        <string>#{opt_bin}/minio</string>
+        <string>server</string>
+        <string>--config-dir=#{CONFIGDIR}</string>
+        <string>--address ":9000"</string>
```
Comment (MikeMcQuaid) [inline Formula/minio.rb]: Are you sure the `"` are needed?

**Comment by MikeMcQuaid** [comment]
Push any changes to this branch and the PR will be updated (rather than opening a new one)

**Comment by opteryxl** [comment]
remove extra quotes

**Comment by opteryxl** [comment]
inline vars

File: Formula/minio.rb [hunk 3]
```diff
@@ -41,6 +41,41 @@ def install
     bin.install buildpath/"minio"
   end
 
+  def post_install
+    datadir = var/"minio".freeze
+    configdir = etc/"minio".freeze
+    datadir.mkpath
```
Comment (MikeMcQuaid) [inline Formula/minio.rb]: `(var/"minio").mkpath` instead

**Comment by MikeMcQuaid** [comment]
One more change then good to go 👍

**Comment by opteryxl** [comment]
ok, changed

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**